### PR TITLE
update Makefile to include local bin operator-sdk

### PR DESCRIPTION
**Pull request by yazug**
specify version for operator-sdk for make bundle
download operator-sdk pinned version when generating bundle update make bundle to use local operator-sdk
update to 1.31 version

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/designate-operator
-  newTag: latest
+  newName: registry-proxy.engineering.redhat.com/rh-osbs/rhosp-dev-preview-trunk-designate-rhel9-operator
```
Comment (viroel) [inline config/manager/kustomization.yaml:15]: @yazug , is this needed?

Comment (yazug) [reply config/manager/kustomization.yaml]: no not needed, reverted that change and rebased to current. Thanks for spotting that.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**